test/actions/update.spec.js

```diff
@@ -175,7 +175,44 @@ describe('Update action creator', () => {
     };
 
     const item = 0;
-    expect(() => update(schemaConfig, item)).to.throw('Item isn\'t object.');
+    expect(() => update(schemaConfig, item)).to.throw('Item is not valid in method argument');
+  });
+
+  it('does not throw exception on action with missing item', () => {
+    const config = {
+      headers: {
+        'Content-Type': 'application/vnd.api+json',
+      },
+      endpoint: 'api.test',
+    };
+
+    const schema = 'app.builder';
+    const schemaConfig = {
+      schema,
+      request: config,
+    };
+    expect(() => update(schemaConfig))
+      .to.not.throw('Item is missing in method argument and in config.body');
+  });
+
+  it('uses body from config if the item is missing', () => {
+    const config = {
+      headers: {
+        'Content-Type': 'application/vnd.api+json',
+      },
+      body: 'Body',
+      endpoint: 'api.test',
+    };
+
+    const schema = 'app.builder';
+    const schemaConfig = {
+      schema,
+      request: config,
+    };
+
+    const action = update(schemaConfig);
+
+    expect(action[RSAA].body).to.equal(config.body);
```

Comment:
Add test that tests what if I pass item and body. Do it please for create and update.